java/client/src/main/java/glide/api/commands/ServerManagementClusterCommands.java

```diff
@@ -22,9 +22,11 @@ public interface ServerManagementClusterCommands {
      * @return Response from Redis cluster with a <code>Map{@literal <String, String>}</code> with
      *     each address as the key and its corresponding value is the information for the node.
      * @example
-     *     <p><code>
-     *     {@literal Map<String, String>} routedInfoResult = clusterClient.info().get().getMultiValue();
-     *     </code>
+     *     <pre>{@code
+     * ClusterValue<String> payload = clusterClient.info().get();
+     * assert payload.getMultiValue().get("node1.example.com:6379").contains("# Stats");
+     * assert payload.getMultiValue().get("node2.example.com:6379").contains("# Stats");
```

Comment:
in each example, document that the choice of `getMultiValue`/`getSingleValue` is determined either by the passed route, or by the function's defaults. it's clear to us, but wouldn't be to newcomers.

Reply:
We do have checks for hasMultiData or hasSingleData would you prefer something like 
```
     ClusterValue<String> payload = clusterClient.info().get();
     if (payload.hasMultiData(){
     assert payload.getMultiValue().get("node1.example.com:6379").contains("# Stats");
     assert payload.getMultiValue().get("node2.example.com:6379").contains("# Stats");

Reply:
> document that the choice of getMultiValue/getSingleValue is determined either by the passed route, or by the function's defaults

again, you need to explain to the reader what happens. adding `if`s doesn't offer an explanation. please write something like "// by default `info` is routed to all nodes in the cluster, hence we assume that the result is a multi value", or "// we route this request to a single random node, so we assume that the result is a single value". and similar to other commands.

Reply:
Okay will do thanks for the feedback.

Reply:
Updated documentation! If you like what I have done I'll apply it to all cluster commands.